linkerd-tcp/Dockerfile

```diff
@@ -1,3 +1,10 @@
-FROM golang:1.8-onbuild
+FROM golang:1.10.3 as builder
+WORKDIR /go/src/github.com/linkerd/linkerd-examples/linkerd-tcp/
+RUN go get -d -v github.com/go-redis/redis
+COPY web.go .
+RUN CGO_ENABLED=0 GOOS=linux go build -o app .
 
-ENTRYPOINT ["app"]
+FROM alpine:3.8
```

Comment:
TIOLI, though these images have `wget`, having `curl` available in these images can be helpful in debugging.

Reply:
Good call -- have added that in the latest commit. I also converted the influxdb and failure-accrual dockerfiles.